src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,54 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.0
+
+**New features**
+
+* Added a new `subctl uninstall` command that removes all Submariner components and dataplane artifacts, such as IP table rules, routes from
```

Comment:
Instead of starting with "subctl uninstall", can we move it to Line 30 ;-)

Reply:
I think its an important and much needed feature addition and should be highlighted :P

Reply:
Haha...I'm not against highlighting :D 

I felt that its like showing how to uninstall your product even before talking about the product features. They should generally be listed at the end (unless there are no other important things to highlight) ;-) :P

Reply:
Anyways, this is more of a nit and I'm fine if others are okay with the current flow.